src/app/Http/Controllers/API/ChoiceController.php

```diff
@@ -24,9 +27,13 @@ class ChoiceController extends Controller
      *
      * @param Poll $poll
      */
-    public function index(Poll $poll)
+    public function index(Request $request, Poll $poll)
     {
-        //
+        if (!PollController::getRequestUser($request, $poll)) {
+            return response()->json('unauthorized', Response::HTTP_UNAUTHORIZED);
+        }
+
+        return ChoiceResource::collection($poll->choices);
```

Comment:
you should also check the status of the poll as normal users can't see a pool that is in a draft status

Reply:
Fixed as per last commit